heartpoints.sh

```diff
@@ -53,6 +64,10 @@ heartpoints_yarn() { local args=$@
     yarn ${args}
 }
 
+heartpoints_runWebPackDevServer(){
+    heartpoints_yarn webpack-dev-server --hot --inline --color --open
```

Comment:
a `heartpoints_yarn install` before this line would be useful :)

Reply:
Some more context: I didn't have `webpack-dev-server` installed so when I ran `./hp runWebPackDevServer` i got an error saying it didn't exist. You didn't run into this issue since you installed it manually with `yarn add webpack-dev-server` when you were working on this code change with Tomtom

Reply:
Understood! What I'm going to do is include a call to heartpoints_prepareForRun inside of heartpoints_clientDev before the call to hearpoints_runWebpackDevServer.

This will ensure that all files are recompiled by the call to webpack. As we were experimenting, we realized that webpack.config.js wasn't being rebuilt (in the event of a change to it) when the dev server was launched, so webpack would have to be run prior to launching webpack-dev-server. Calling prepareForRun will amend this issue, as well as the issue mentioned above. 

Pushing now. Thanks, guys!

Reply:
calling `./hp clientDev` will now install all dependencies, rebuild the entire project, and launch webpack dev server.